<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSpreadsheetWidget</class>
 <widget class="QWidget" name="ExportSpreadsheetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>672</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbExport">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lFileName">
        <property name="text">
         <string>File name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="leFileName">
        <property name="toolTip">
         <string>Specify the name of the file to import.</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="bOpen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string> Select the file to import</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="cbFormat"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lFormat">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lExportHeader">
        <property name="text">
         <string>Export header:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkExportHeader">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lMatrixVHeader">
        <property name="text">
         <string>Export vertical header:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkMatrixVHeader">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lMatrixHHeader">
        <property name="text">
         <string>Export horizontal header:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="chkMatrixHHeader">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lSeparator">
        <property name="text">
         <string>Separating string:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KComboBox" name="cbSeparator">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lDecimalSeparator">
        <property name="text">
         <string>Decimal separator:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cbDecimalSeparator"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lGridLines">
        <property name="text">
         <string>Grid lines:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="chkGridLines">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lCaptions">
        <property name="text">
         <string>Table captions:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="chkCaptions">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lHeader">
        <property name="text">
         <string>Headers:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="chkHeaders">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lEmptyRows">
        <property name="text">
         <string>Skip empty rows:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="chkEmptyRows">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lExportArea">
        <property name="text">
         <string>Area to export:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="cbLaTeXExport"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lExportToFITS">
        <property name="text">
         <string>Export to:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="cbExportToFITS"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lColumnAsUnits">
        <property name="text">
         <string>Comments as column units:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QCheckBox" name="chkColumnsAsUnits">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
